source/aggregation-pipeline-builder.txt

```diff
@@ -73,8 +73,62 @@ Limitations
 Create an Aggregation Pipeline
 ------------------------------
 
+Aggregation pipelines can be created in 2 seperate modes:
+
+- Stage View Mode, a visual click based pipeline editor which 
+  preloads pipeline syntax based on user selections.
+
+- Text View Mode, a text based pipeline editor which accepts raw
+  pipeline syntax. 
```

Comment:
I think for both of these "which" should be "that".

I'm loosing basing this recommendation off of this [article](https://www.diffen.com/difference/That_vs_Which), which states that generally when we use "which" it should appear after a comma.